### PR TITLE
add waldo directory

### PR DESCRIPTION
**Pull request by nrhubbar**
@herrejes I created this because we should be merging into Dev first then master

### CODE REVIEW

File: waldo/app/configs/mongodb.config.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+var blueprint = require ('@onehilltech/blueprint')
+  ;
+
+module.exports = {
+  connections: {
+    $default: {
+      connstr: 'mongodb://localhost/waldo_' + blueprint.env,
```
Comment (nrhubbar) [inline waldo/app/configs/mongodb.config.js]: You will want to change `localhost` to the ip of  mongo:
`165.227.76.52:32768`

File: waldo/app/controllers/HelloWorldController.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+var blueprint = require ('@onehilltech/blueprint')
```
Comment (nrhubbar) [inline waldo/app/controllers/HelloWorldController.js]: We will want to get rid of this file, but it does not have to happen this PR _"pull request"_

**Comment by nrhubbar** [comment]
Looks good, I assume it runs fine on your machine.

Make the changes I suggested and we should be good!

**Comment by herrejes** [comment]
Yes it ran fine on my machine.  I will make these changes ASAP.  Thanks for the help.

**Pull request merged**